### PR TITLE
feat: 🎸 Switch dashboard view feature

### PR DESCRIPTION
**Pull request by dariuszlacheta**

### CODE REVIEW

File: src/components/DashboardViewSwitch/DashboardViewSwitch.styles.ts [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  border-top: 1px solid ${colors.gray[300]};
+  border-bottom: 1px solid ${colors.gray[300]};
+`;
+
+export const List = styled.ul`
+  padding: 0;
+  margin: 0;
+  list-style: none;
+  font-size: 16px;
+  font-family: Lato Medium, sans-serif;
```
Comment (apryka) [inline src/components/DashboardViewSwitch/DashboardViewSwitch.styles.ts:69]: please update to `Lato Regular`

File: src/components/DashboardViewSwitch/DashboardViewSwitch.test.tsx [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+test('renders dashboard switch with dropdown and allows users to search dashboard based on phrase', () => {
+  const {
+    wrapper: { container, getAllByTestId, getByText },
+  } = render();
+
+  const dashboardSwitch = getByText('dashboard_details.untitled_dashboard');
+  fireEvent.click(dashboardSwitch);
+
+  waitFor(() => {
+    const searchInput = container.querySelector('input[type="text"]');
```
Comment (apryka) [inline src/components/DashboardViewSwitch/DashboardViewSwitch.test.tsx:83]: `getByRole('textbox')` should be used instead

File: src/components/DashboardViewSwitch/DashboardViewSwitch.tsx [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+          <DropIndicator>
+            <Icon
+              type="caret-down"
+              fill={colors.blue[500]}
+              width={15}
+              height={15}
+            />
+          </DropIndicator>
+        </TitleWrapper>
+        <DropdownContainer style={{ zIndex: UI_LAYERS.dropdown }}>
```
Comment (apryka) [inline src/components/DashboardViewSwitch/DashboardViewSwitch.tsx:126]: please move `zIndex` to styles.tsx

File: src/components/DashboardViewSwitch/DashboardViewSwitch.styles.ts [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+`;
+
+export const DropdownFooter = styled.div`
+  display: fix;
```
Comment (maciejrybaniec) [inline src/components/DashboardViewSwitch/DashboardViewSwitch.styles.ts:73]: I believe it should be `display: flex`

**Review by maciejrybaniec — APPROVED** [review]

File: src/components/DashboardViewSwitch/DashboardViewSwitch.tsx [hunk 5]
```diff
@@ -0,0 +1,185 @@
...
+      const phrase = searchPhrase.toLowerCase();
+      dashboardsList = dashboardsList.filter(
+        ({ title }) => title && title.toLowerCase().includes(phrase)
+      );
+    }
+
+    return dashboardsList;
+  }, [searchPhrase, dashboards]);
+
+  const createDashbord = useCallback(() => {
```
Comment (apryka) [inline src/components/DashboardViewSwitch/DashboardViewSwitch.tsx:75]: typo `createDashboard`

**Review by apryka — APPROVED** [review]

**Pull request merged**